### ISSUE TITLE
Limit printed output

### ISSUE THREAD

**Issue by antonia-had** [OpenCDSS/cdss-app-statemod-fortran#28]
Hi,
This is less of an issue and more of a question. Is there are way to either limit the printed output or eliminate it when running the Simulation module? I'm referring to what is printed out as the model is running, not the reports generated at the end. 
I'm running thousands of simulations in parallel and I end up having millions of lines of 
```
Execut; Year  1994 Month NOV  Day   1 Reoperation     1 Annual Total     1
Execut; Year  1996 Month JUL  Day   1 Reoperation     1 Annual Total     9
Execut; Year  1990 Month JUN  Day   1 Reoperation     1 Annual Total     8
```
in my output file, which makes it very large. 

I'm running StateMod version 15.00.01

**Comment by macphersonbr** [comment]
Commenting out lines execut.for line 812 (write(6,103) iyrmo(mon), xmonam(mon), idy, iwx, iwxmaxY) and mdainp.for line 112 (write(nscrn,*) ' Mdainp.for') should eliminate the looped output. Hope that helps!

**Comment by antonia-had** [comment]
Thanks! That worked!

**Comment by smalers** [comment]
We should enhance StateMod to have user-controlled output levels.  This can use standard "word" levels like "critical", "warn", "info", "debug", "trace", or a numerical level (bigger number means more input).  There is some of this built into StateMod already but it could be enhanced.  I am going to add a new issue to focus on this.
